### PR TITLE
feat: add support for anytrust orbit chain

### PR DESCRIPTION
**Pull request by GreatSoshiant**
In this Pull Request, we introduce modifications to the Docker Compose configuration, enabling the seamless execution of the DAS-Server service specifically for AnyTrust Orbit Chains. Notably, if an Orbit Chain is identified as a rollup and not AnyTrust, the data availability server will enter an indefinite sleep state.

A new bash script has been incorporated into the DAS-Server's entrypoint. This script is responsible for feeding the Sequencer Inbox Contract to the command-line interface of the AnyTrust DAS server from nodeConfig.json provided by users. Crucially, this bash script ensures that both AnyTrust and Rollup deployments are handled correctly, adhering to the operational requirements previously discussed.

### CODE REVIEW

File: docker-compose.yaml
Comment (gzeoneth) [inline docker-compose.yaml:1]: seems to have some unnecessary indent in the file

File: config/das-server.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# Read sequencerInbox address from the JSON file
+SEQUENCER_INBOX_ADDRESS=$(jq -r '.sequencerInbox' /home/user/.arbitrum/orbitSetupScriptConfig.json)
+
+# Check if the "data-availability" key exists
+if jq -e '.node | has("data-availability")' /home/user/.arbitrum/nodeConfig.json >/dev/null; then
+    # Start daserver with the new command
+    /usr/local/bin/daserver --data-availability.parent-chain-node-url https://goerli-rollup.arbitrum.io/rpc --enable-rpc --rpc-addr '0.0.0.0' --enable-rest --rest-addr '0.0.0.0' --log-level 3 --data-availability.local-db-storage.enable --data-availability.local-db-storage.data-dir /home/user/data/badgerdb --data-availability.local-db-storage.discard-after-timeout --data-availability.key.key-dir /home/user/.arbitrum/keys --data-availability.local-cache.enable --data-availability.sequencer-inbox-address $SEQUENCER_INBOX_ADDRESS
+else
+    # No-op (do nothing) if "data-availability" key doesn't exist
+    sleep infinity
```
Comment (gzeoneth) [inline config/das-server.sh:12]: I think you don't need to make it sleep here you can just let it return?

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,6 @@ cache
 artifacts
 config/*
 .DS_Store
-outputInfo.json
\ No newline at end of file
+outputInfo.json
+config/.DS_Store
+!config/das-server.sh
```
Comment (gzeoneth) [inline .gitignore:8]: instead of doing this, you can put the script in root and mount it separately

Comment (GreatSoshiant) [reply .gitignore:8]: Done! Thanks ser!

File: docker-compose.yaml [hunk 3]
```diff
@@ -25,21 +25,31 @@ services:
...
+      volumes:
+        - "./config:/home/user/.arbitrum"
+      command: --conf.file /home/user/.arbitrum/nodeConfig.json
+
+    das-server:
+      image: offchainlabs/nitro-node:v2.1.0-beta.6-152b9ff
+      entrypoint: ["/bin/bash", "/main/das-server.sh"]
+      volumes:
+          - "./config:/home/user/.arbitrum"
+          - ".:/main"
```
Comment (gzeoneth) [inline docker-compose.yaml:52]:
```suggestion
      entrypoint: ["/bin/bash", "/das-server.sh"]
      volumes:
          - "./config:/home/user/.arbitrum"
          - "./das-server.sh:/das-server.sh"
```

File: config/das-server.sh (hunk 1 above)
Comment (GreatSoshiant) [reply config/das-server.sh]: Completely makes sense! Changed it to exit, and add a log for exiting!

**Review by gzeoneth — APPROVED** [review]
LGTM

**Pull request merged**